packages/core/sdk/src/auth/internal/CoreStitchAuth.ts

```diff
@@ -211,6 +211,10 @@ export default abstract class CoreStitchAuth<TStitchUser extends CoreStitchUser>
   ): Promise<T> {
     return this.doAuthenticatedRequest(stitchReq)
       .then(response => {
+        if (response.body! === '{"$undefined":true}') {
+          return undefined;
+        }
+       
         const obj = EJSON.parse(response.body!, { strict: false });
```

Comment:
This is what I added, let me know if this is too sketchy

Reply:
This is very sketchy, and I don't think it's a check we should be making for every request. 

Would it be better to do the following

```
if (obj === null) {
    return undefined;
}
```

after the EJSON.parse call?